blog/_posts/2017-8-25-gpuarrays.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Julia's GPU ecosystem: A status repot"
+author: <a href="https://github.com/SimonDanisch">Simon Danisch</a>
+---
+
+A lot has happening lately Julia's GPU ecosystem.
+[Tim Maleadt](https://github.com/maleadt) has put great effort into making [CUDAnative.jl](https://github.com/JuliaGPU/CUDAnative.jl/) a fully fledged
+compiler backend to execute Julia code on nvidia GPUs.
+
+[Mike Innes](https://github.com/MikeInnes) just published [CuArrays](https://github.com/FluxML/CuArrays.jl) along with a detailed [blog post](http://mikeinnes.github.io/2017/08/24/cudanative.html) outlining all the great features Julia brings into the world of GPU programming.
+
+I want to add some more words about my recent work on [GPUArrays](https://github.com/JuliaGPU/GPUArrays.jl) and benchmarking our current GPU packages.
+
+GPUArrays is a backend independent N dimensional Array package that builds upon the work
+of [Transpiler.jl](https://github.com/SimonDanisch/Transpiler.jl) for an OpenGL and OpenCL backend and uses [CUDAnative](https://github.com/JuliaGPU/CUDAnative.jl/) for nvidia based hardware.
+It also offers a threaded Julia backend which can even utilize Xeon Phi's and is great for debugging the GPU kernels.
+GPUArrays also offers abstractions over the intrinsics of the different backends and makes it possible to write GPU programs
+completely in Julia for all hardware.
+
+CuArrays is very similar to GPUArrays cudanative backend and was first supposed to be an extension of that,
+but we decided to concentrate on CUDA support in it's own package to create a much leaner package.
+This is a fair approach for now and we plan to merge things back at some point to decrease fragmentation of the ecosystem.
+
+Like CuArrays, GPUArrays supports functions like [broadcast](https://julialang.org/blog/2017/01/moredots), `mapreduce`, indexing, `vcat`/`hcat` and all functionality that comes for free from the support of these generic functions, e.g. sum, maximum, etc.
```

Comment:
The code formatting applied to function names is inconsistent here. I'd add backticks to broadcast, sum, and maximum.

Reply:
thanks!

Reply:
Once that's addressed I think this is good to go!

Reply:
oh i misunderstood and thought you corrected it already^^